dev/create-release/release-build.sh

```diff
@@ -333,7 +333,13 @@ if [[ "$1" == "docs" ]]; then
   echo "Building Spark docs"
   cd docs
   # TODO: Make configurable to add this: PRODUCTION=1
-  PRODUCTION=1 RELEASE_VERSION="$SPARK_VERSION" jekyll build
+  if [ ! -f "Gemfile" ]; then
```

Comment:
As release script can be used to build any Spark refs (see GIT_REF) where could be no Gemfile at all.